data/json/monstergroups/eggs.json

```diff
@@ -495,5 +495,75 @@
     "name": "GROUP_EGG_FALLFISH",
     "type": "monstergroup",
     "monsters": [ { "monster": "mon_fry_fallfish" } ]
+  },
+  {
+    "name": "GROUP_EGG_PROXY_FROG",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_proxy_tadpole" } ]
+  },
```

Comment:
Is this intended to exist? ``proxy_frog`` looks to be your abstract, why does it have functional egg mongroups?